src/main/java/edu/kis/powp/jobs2d/drivers/adapter/MyAdapter.java

```diff
@@ -4,15 +4,19 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
 public class MyAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
Name should be fixed.
Inheritance is not used (no super drawLine calls). Should be removed!